src/reconcile/rec_visibility.c

```diff
@@ -19,12 +19,13 @@ __rec_update_save(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, WT_
     WT_SAVE_UPD *supd;
 
     /* If nothing is committed, we must restore the update chain. */
-    WT_ASSERT(session, onpage_upd != NULL || supd_restore);
+    WT_ASSERT_STRONG(session, onpage_upd != NULL || supd_restore);
     /* We can only write a standard update or a modify to the data store. */
-    WT_ASSERT(session,
+    WT_ASSERT_STRONG(session,
       onpage_upd == NULL || onpage_upd->type == WT_UPDATE_STANDARD ||
         onpage_upd->type == WT_UPDATE_MODIFY);
     /* For columns, ins is never null, so rip == NULL implies ins != NULL. */
+    // WT-9417 IGNORE - Column store
```

Comment:
Why some of them have WT, some not?

Reply:
That's just a typo. These are temporary comments to highlight unchanged asserts in case they should be upgraded, and will be removed before merge.